### ISSUE TITLE
llvmXX fail on 10.14 with fat building

### ISSUE THREAD

**Issue by nieder** [fink/fink-distributions#340]
The llvmXX packages try to build both i386 and x86_64 fat binaries but this is no longer a working model on 10.14:
````
    Run Build Command:"/usr/bin/make" "cmTC_36503/fast"
    /Library/Developer/CommandLineTools/usr/bin/make -f CMakeFiles/cmTC_36503.dir/build.make CMakeFiles/cmTC_36503.dir/build
    Building CXX object CMakeFiles/cmTC_36503.dir/testCXXCompiler.cxx.o
    /sw/src/fink.build/llvm50-5.0.0-1/opt-bin/cc-st1-clang++    -arch i386 -arch x86_64  -arch x86_64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -mmacosx-version-min=10.14   -o CMakeFiles/cmTC_36503.dir/testCXXCompiler.cxx.o -c /sw/src/fink.build/llvm50-5.0.0-1/build/stage1.5/CMakeFiles/CMakeTmp/testCXXCompiler.cxx
    Linking CXX executable cmTC_36503
    /sw/bin/cmake -E cmake_link_script CMakeFiles/cmTC_36503.dir/link.txt --verbose=1
    /sw/src/fink.build/llvm50-5.0.0-1/opt-bin/cc-st1-clang++   -arch i386 -arch x86_64  -arch x86_64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -mmacosx-version-min=10.14 -Wl,-search_paths_first -Wl,-headerpad_max_install_names -L/sw/lib  CMakeFiles/cmTC_36503.dir/testCXXCompiler.cxx.o  -o cmTC_36503 
    ld: warning: The i386 architecture is deprecated for macOS (remove from the Xcode build setting: ARCHS)
    ld: warning: ignoring file /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk/usr/lib/libSystem.tbd, missing required architecture i386 in file /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk/usr/lib/libSystem.tbd
    ld: warning: ignoring file /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk/usr/lib/libc++.tbd, missing required architecture i386 in file /Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk/usr/lib/libc++.tbd
    ld: dynamic main executables must link with libSystem.dylib for architecture i386
    clang-5.0: error: linker command failed with exit code 1 (use -v to see invocation)
    make[1]: *** [cmTC_36503] Error 1
````
@jwhowarth

**Comment by nieder** [comment]
This will need similar changes as gccX. Namely, removing i386 building, and an extra change to set CMAKE_OSX_SYSROOT to $SDK_PATH. Local test build has made it to stage 2.

**Comment by dmacks** [comment]
Is this now resolved? For example, abe0c28ee9db82fac5b058221632e95273d42112 for llvm50 claims to be x86_64-only on 10.14+.

**Comment by nieder** [comment]
Yes. We need to make new .info for 10.14.5 and 10.15.
